.mailmap

```diff
@@ -0,0 +1,48 @@
+Amedeo Sapio <asapio@amazon.com>
+Arun Karthik <akkart@amazon.com>
+Aviv Benchorin <benchori@amazon.com>
+Brian Barrett <bbarrett@amazon.com>
+David Addison <daddison@nvidia.com>
+Denis Maryin <denis.maryin@intel.com>
+Dmitry Gladkov <dmitry.gladkov@intel.com>
+Dmitry Monakhov <monakhov@amazon.com>
+Eric Raut <eraut@amazon.com>
+Gargi Saha <grgisaha@amazon.com>
+Greg Inozemtsev <ginozemt@amazon.com>
+Hunter North <hunnorth@amazon.com>
+James Dinan <jdinan@nvidia.com>
+James Siri <jamesiri@amazon.com>
+Liran Alon <liran@amazon.com>
+Michael Axtmann <axtmannm@amazon.com>
+Nafea Bshara <nafea@amazon.com>
+Nicholas Sielicki <nslick@amazon.com>
+Raghu Raja <raghunch@amazon.com>
+Rashika Kheria <rashika@amazon.com>
+Ryan Hankins <ryan.hankins@hpe.com>
+Sai Sunku <sunkusa@amazon.com>
+Se Wang Oh <seoh@amazon.com>
+Seth Zegelstein <szegel@amazon.com>
+Shi Jin <sjina@amazon.com>
+Sylvain Jeaugey <sjeaugey@nvidia.com>
+Taeil Um <taeilum@amazon.com>
+Thomas Huber <thomas.huber@cornelisnetworks.com>
+Venky Natham <vrnatham@amazon.com>
+Verdi March <marcverd@amazon.com>
+Yongseok Koh <koyongse@amazon.com>
+YoonGi Kim <yoongi@amazon.com>
+ilya minkin <imminkin@amazon.com>
+<akkart@amazon.com> <114260793+arunkarthik-akkart@users.noreply.github.com>
+<bbarrett@amazon.com> <ubuntu@ip-10-192-10-147.ec2.internal>
+<daddison@nvidia.com> <david.addison@gmail.com>
+<eraut@amazon.com> <10216922+rauteric@users.noreply.github.com>
+<eraut@amazon.com> <eric@badgerio.us>
+<eraut@amazon.com> <rauteric@users.noreply.github.com>
+<hunnorth@amazon.com> <119898856+hunnorth@users.noreply.github.com>
+<monakhov@amazon.com> <dmonakhov@gmail.com>
+<raghunch@amazon.com> <craghun@amazon.com>
+<raghunch@amazon.com> <raghu@enfabrica.net>
```

Comment:
I'm totally fine with most of the cleanups (and a little embarassed that somewhere I didn't reset that author for the ubuntu@... entry).  Fixing Raghu's email change at amazon also makes sense.  But I don't think we want to change email addresses across companies.  The copyright for most of the changes are really owned by the company, not the person, so we shouldn't be rewriting that history.  This is most notable for Raghu and Ryan.